src/coverlet.console/Program.cs

```diff
@@ -172,13 +172,17 @@ static int Main(string[] args)
                 var totalBranchPercent = summary.CalculateBranchCoverage(result.Modules).Percent;
                 var totalMethodPercent = summary.CalculateMethodCoverage(result.Modules).Percent;
 
+                var averageLinePercent = summary.CalculateLineCoverage(result.Modules).AverageModulePercent;
+                var averageBranchPercent = summary.CalculateBranchCoverage(result.Modules).AverageModulePercent;
+                var averageMethodPercent = summary.CalculateMethodCoverage(result.Modules).AverageModulePercent;
```

Comment:
We can reuse objects from line# 171, 172 and 173, then get total percent and average percent from those `CoverageDetails` object for each type line, branch and module respectively. This way we avoid doing recalculation and get some perf benefit. What you think?

Reply:
yep, it need a refactoring